charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```
@@ -21,9 +21,9 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/cmd-params: {{ include (print $.Template.BasePath "/argocd-configs/argocd-cmd-params-cm.yaml") . | sha256sum }}
+        checksum/cmd-params: {{ (include (print $.Template.BasePath "/argocd-configs/argocd-cmd-params-cm.yaml") . | fromYaml).data | toYaml | sha256sum }}
```

What about 
```suggestion
        checksum/cmd-params: {{ include "argo-cd.config.params" . | sha256sum }}
```
Ref: https://github.com/argoproj/argo-helm/blob/936101865d3090183fc4979546685ece7f78ac23/charts/argo-cd/templates/argocd-configs/argocd-cmd-params-cm.yaml#L16